payments/invoice/other-features.md

```diff
@@ -233,53 +798,121 @@ Content-Type: application/json
+{% include one-click-payments.md %}
 
-When performing unsuccessful operations, the eCommerce API will respond with
-a problem message. We generally use the problem message type and status code
-to identify the nature of the problem. The problem name and description will
-often help narrow down the specifics of the problem.
+{% include callback-reference.md %}
 
-#### Error types from Swedbank Pay Invoice and third parties
+{% include payment-link.md %}
 
-All invoice error types will have the following URI in front of type:
-`https://api.payex.com/psp/errordetail/invoice/<errorType>`
+{% include subsite.md %}
 
-{:.table .table-striped}
-| **Type**        | **Status** |
-| :-------------- | :--------- |
-| *externalerror* | 500        | No error code                 |
-| *inputerror*    | 400        | 10 - ValidationWarning        |
-| *inputerror*    | 400        | 30 - ValidationError          |
-| *inputerror*    | 400        | 3010 - ClientRequestInvalid   |
-| *externalerror* | 502        | 40 - Error                    |
-| *externalerror* | 502        | 60 - SystemError              |
-| *externalerror* | 502        | 50 - SystemConfigurationError |
-| *externalerror* | 502        | 9999 - ServerOtherServer      |
-| *forbidden*     | 403        | Any other error code          |
+## PayeeReference
 
-{% include settlement-reconciliation.md %}
+{% include payeeinfo.md %}
 
-{% include payment-link.md %}
+## Prices
 
-{% include callback-reference.md %}
+{% include prices.md %}
 
-## Payment Orders
+{% include settlement-reconciliation.md %}
 
-{% include payment-order-get.md %}
+## Problems
 
-{% include transactions-reference.md %}
+When performing unsuccessful operations, the eCommerce API will respond with a
+problem message. We generally use the problem message `type` and `status` code
+to identify the nature of the problem. The problem `name` and `description` will
+often help narrow down the specifics of the problem.
 
-{% include operations-reference.md %}
+### Error types from Swedbank Pay Invoice and third parties
 
-### Payee Info
+All invoice error types will have the following URI in front of type:
+`https://api.payex.com/psp/errordetail/invoice/<errorType>`
 
-{% include payeeinfo.md %}
+{:.table .table-striped}
+| Type            | Status |                               |
+| :-------------- | :----- | :---------------------------- |
+| *externalerror* | 500    | No error code                 |
+| *inputerror*    | 400    | 10 - ValidationWarning        |
+| *inputerror*    | 400    | 30 - ValidationError          |
+| *inputerror*    | 400    | 3010 - ClientRequestInvalid   |
+| *externalerror* | 502    | 40 - Error                    |
+| *externalerror* | 502    | 60 - SystemError              |
+| *externalerror* | 502    | 50 - SystemConfigurationError |
+| *externalerror* | 502    | 9999 - ServerOtherServer      |
+| *forbidden*     | 403    | Any other error code          |
+
+### Acquirer and 3-D Secure Problem Types
```

Comment:
Please remove this section as acquirer and 3D secure is not supposed to be in Invoice Payments